### DISCUSSION TITLE
I'm new to GitHub

### DISCUSSION THREAD

**Discussion by Broken-03** [community/community#44316] [New to GitHub]
Trying to learn how to build my first respiratory and get copilot if anyone could help me out

**Comment by radar-blip** [comment]
Here's a link for [creating a repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-new-repository)

And here's a source for [copilot](https://github.com/features/copilot)

**Reply by Broken-03** [reply]
Thank you but I never done this before the reason I am now because my phone is acting up and because I think some one put some kind of coding or something

**Comment by Broken-03** [comment]
AM David thank you
